patchwork-events-entity/src/main/java/net/patchworkmc/mixin/event/entity/MixinExperienceOrbEntity.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Minecraft Forge, Patchwork Project
+ * Copyright (c) 2016-2020, 2019-2020
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation version 2.1
+ * of the License.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+package net.patchworkmc.mixin.event.entity;
+
+import net.minecraftforge.common.MinecraftForge;
+import net.minecraftforge.event.entity.player.PlayerPickupXpEvent;
+import org.objectweb.asm.Opcodes;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
+
+import net.minecraft.entity.player.PlayerEntity;
+import net.minecraft.entity.ExperienceOrbEntity;
+
+@Mixin(ExperienceOrbEntity.class)
+public class MixinExperienceOrbEntity {
+	// After checking we're on the server and the player is ready to pick up the orb, the first
+	// thing the target method does is set experiencePickUpDelay, hence hook just before that.
+	@Inject(method = "onPlayerCollision", cancellable = true, at = @At(value = "FIELD", opcode = Opcodes.H_PUTFIELD, ordinal = 0,
```

Comment:
This Handle opcode looks suspicious. Have you tested this mixin and confirmed it works?

Reply:
While you're at it please make sure this launches in a reobf environment (compiled and in MultiMC/vanilla launcher)

Reply:
I've tested it in a dev environment, I'll test it in MultiMC now.

Reply:
Looking at the patch this seems to be identical.

Reply:
As in the commit I just pushed? That converted some comments to a JavaDoc comment

Reply:
Clarified over discord that was referring to the bytecode matching.

Reply:
Since I didn't set it up properly I thought it wasn't working in reobf, tested it now and it works fine as expected.